### ISSUE TITLE
GlobalTwitchEmotes and Edge support.

### ISSUE THREAD

**Issue by ghost** [melalawi/GlobalTwitchEmotes#11]
Hey!
So I built my GlobalTwitchEmotes extension for Edge, and it works really good! tested it against Chrome version and it has the same issues, like I don't know if the refresh icon is suppose to be a button or it is an indication that I need to refresh.
But I noticed when you whitelist or blacklist then it appears the refreshing icon and the gear settings icon doesn't work there. so the extension has to be clicked away and then back again in order to go to settings or something, and this happens on both chrome and edge.

On edge the export button doesn't work.

But everything else works great, it didn't take me long to build it for edge and load it, but what is the reason isn't yet on store? it would be great to have it officially on store and make the extension page larger for us Edge users.

But whitelist and blacklist works great (domains, webpages, channels and emotes), it shows all possible emotes if offers, I tested it and it doesn't seem to slow down Edge much different than anything else would.

So this is like a ticket to say some issues but also but also request if you can make it available on store. Works just as fine as Chrome for sure (minus the export button that not many would care about)

Thank you and have a great day!.

**Comment by melalawi** [comment]
Hey ghost.

First off, very sorry about the delay.

So, the last time I worked on this, the only way one could publish their extension to the Windows Store was if they had received an invite from Microsoft. It seems this is still the case. I resubmitted a request for said invite [here](aka.ms/extension-request).

> I don't know if the refresh icon is suppose to be a button or it is an indication that I need to refresh.

It's supposed to be a working refresh button. As far as I can tell, the BrowserAction element seems to be broken in many ways across all browsers. I'll look into this.

> On edge the export button doesn't work.

I'll look into this soon.

Thank you.
